File: ExamApi/Models/Candidate.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
public class Candidate
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Email { get; set; } = string.Empty;

        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string? Phone { get; set; }

        public ICollection<CandidateExam> CandidateExams { get; set; } = new List<CandidateExam>();
        public ICollection<CandidateAnswer> CandidateAnswers { get; set; } = new List<CandidateAnswer>();
    }

File: Frontend/exam.ui/types/candidate.ts
export interface CandidateAnswer {
 questionText: string;
  questionImageUrl?: string;
  choiceText: string;
  choiceImageUrl?: string;
   isCorrect: boolean; 
}

export interface Candidate {
  id: string;
  firstName: string;
  lastName: string;
  email: string;
  phone: string;
  status: number | string | null;
  score: number | null;
  invitedAt: string | null;
  startedAt: string | null;
}
export interface CandidateExam {
  examTitle: string;
  invitedAt: string;
  startedAt: string | null;
  status: number;
  score: number | null;
  answers: CandidateAnswer[];
}

export interface CandidateDetail {
  id: number;
  name: string;
  email: string;
  phone: string;
  exams: CandidateExam[];
}